samples/java_springboot/19.custom-dialogs/src/main/java/com/microsoft/bot/sample/customdialogs/RootDialog.java

```diff
@@ -48,49 +48,64 @@ public RootDialog(UserState withUserState) {
         // Dialogs can be nested and the slot filling dialog makes use of that. In this example
         // some of the child dialogs are slot filling dialogs themselves.
         List<SlotDetails> slots = Arrays.asList(
-            new SlotDetails("fullname", "fullname"),
+            new SlotDetails("fullname", "fullname", null, null),
             new SlotDetails("age", "number", "Please enter your age."),
             new SlotDetails(
                 "shoesize", "shoesize", "Please enter your shoe size.",
                 "You must enter a size between 0 and 16. Half sizes are acceptable."
             ),
-            new SlotDetails("address", "address")
+            new SlotDetails("address", "address", null, null)
```

Comment:
Checking that this will be present, I think that we can rollback the changes of `SlotDetails` as it will make unnecessary nulls here